riscv/tests/riscv_data/evm/src/lib.rs

```diff
@@ -20,10 +21,7 @@ fn main() {
         b256!("e3c84e69bac71c159b2ff0d62b9a5c231887a809a96cb4a262a4b96ed78a1db2");
     let mut db = CacheDB::new(EmptyDB::default());
 
-    let bytecode_len = get_data_len(0);
-    let mut bytecode = vec![0; bytecode_len];
-    get_data(0, &mut bytecode);
-    let bytecode: Vec<u8> = bytecode.into_iter().map(|x| x as u8).collect();
+    let bytecode: Vec<u8> = get_data_serde(666);
```

Comment:
This is the main goal of this PR.

Reply:
Out of curiousity: Is `CODE_HASH` the hash of the prover-provided bytecode? And is this verified? Or is this test just saying "I ran some bytecode", without making a claim about which one?

Reply:
But we could also directly query `AccountInfo` instead of `Vec<u8>`, right?

Reply:
>  Or is this test just saying "I ran some bytecode", without making a claim about which one?

Yes, the test is the first EVM test we ran, so we just kept it but it doesn't really say or prove much.

Reply:
> But we could also directly query `AccountInfo` instead of `Vec<u8>`, right?

yep we could